### DISCUSSION TITLE
aroundLatLngViaIP with React Instasearch/Geo Search

### DISCUSSION THREAD

**Discussion by nathan-schmidt-viget** [algolia/instantsearch#5352] [Q&A] [answered]
I have been trying find docs about how to set the aroundLatLngViaIP for Geo Search using a toggle or just pass it on page load. https://www.algolia.com/doc/api-reference/api-parameters/aroundLatLng/?client=javascript

If you can point me to a docs on how to set that up or even something in codesandbox that would be awesome.

**Comment by Haroenv** [comment, marked as answer]
You can use the Configure widget to set search parameters that aren't interacted by the user

```js
<Configure aroundLatLng="..." />
```
